app/controllers/site.rb

```diff
@@ -90,8 +112,43 @@ class Site < Base
 				end
 			end
 
+			namespace "/:site/users" do
+				get "/" do
+					authorize site, :manage_users?
+					users = Tint.db[:users].where(user_id: site.users.map { |u| u[:user_id] }).map do |user|
```

Comment:
This gets all the user records that are associated with the current site, and then maps over them to add back the "role" from `site.users` to the return records (since the "role" isn't in users, but is in the join table)